CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 5.3.0
+
+* Add an optional `postage` argument to send_precompiled_letter_notification method.
+
```

Comment:
Added postage to the response of send_precompiled_letter_notification

Reply:
> We haven't add postage to integration_test/schemas/v2/notification_schemas/get_notification_response. Should we add it to this PR?

Yes, that's a good idea, I will add it 👍